<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>266</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Repay Debt Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
color:white;
background-color:rgb(54,54,54);
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}
QLineEdit{
border:1px solid grey;
border-radius:5px;
}
QPushButton{
background-color: rgb(0, 170, 255);
border:1px solid grey;
}
QPushButton:hover{
border:2px solid grey;
border-radius:5px;
}
QPushButton:pressed{
border:3px solid orange;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Debt Balance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="costLbl">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>0.00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Amount Paid</string>
         </property>
         <property name="buddy">
          <cstring>amountpaidLE</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="amountpaidLE">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>35</height>
          </size>
         </property>
         <property name="maxLength">
          <number>10</number>
         </property>
         <property name="placeholderText">
          <string>Enter  Amount repaid </string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="repayBtn">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Repay</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
